### PR TITLE
Release/0.4.0

### PR DESCRIPTION
**Pull request by jbeemster**

### CODE REVIEW

File: CHANGELOG [hunk 1]
```diff
@@ -1,3 +1,12 @@
+Version 0.4.0 (2015-11-13)
+--------------------------
+Bump gopkg.in/pg.v2 to v3 (#48)
+Initialize the variables map inside the playbook properly, thanks @andrioni! (#47)
+Add -dryRun mode (#45)
+Add ability for SQL files to be read from Consul (#44)
+Add ability to read playbook from Consul (#43)
+Add an environment variable function (#30)
+
```
Comment (alexanderdean) [inline CHANGELOG]: "Add", "Bump" etc should be in past tense

File: README.md [hunk 2]
```diff
@@ -68,6 +68,24 @@ There are several command line arguments that can be used:
...
+* The `-sqlroot` argument also becomes a key argument for Consul.
+
+Say for example you pass the following command in:
+
+* `./sql-runner -consul "localhost:8500" -playbook "sql-runner/playbook/1" -sqlroot CONSUL`
+
+This results in:
+
+* Looking for your playbook file at this key `sql-runner/playbook/1`
+* Expecting all your SQL file keys to begin with `sql-runner/playbook/1/<SQL path from playbook>`
```
Comment (alexanderdean) [inline README.md]: So `CONSUL` is just a synonym for `PLAYBOOK`? Not sure I follow the logic of introducing a new flag value.

**Comment by andrioni** [comment]
Hi, are you planning to cut and release 0.4.0 this week or the "merge window" is going to be open for a few more days?

**Comment by alexanderdean** [comment]
Probably end of this week!

**Comment by andrioni** [comment]
Nice, thanks!

File: README.md (hunk 2 above)
Comment (jbeemster) [inline README.md]: I added it as a way of having a valid default option when using Consul.  PLAYBOOK and BINARY both look up full paths whereas CONSUL simply keeps the -playbook argument without altering it at all.

**Pull request merged**